src/portfolio_performance_writer.py

```diff
@@ -9,7 +9,7 @@
 import io
 import logging
 
-PP_FIELDNAMES = ['Datum', 'Wert', 'Typ', 'Notiz']
+PP_FIELDNAMES = ['Datum', 'Wert', 'Buchungswährung', 'Typ', 'Notiz']
```

Comment:
Hi Tobi,

Could you have a look at the unit tests?  The addition of this additional field breaks them.  :)

Reply:
Hi Chris,

thanks for the reply. I made the configuration of fee and currency optional and fixed the unit tests. There are checks failing, but for me it looks like project configuration issues which I can't fix.

Reply:
I believe I found the error for the failing unit test coverage upload :( 
https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
that is ... hmm ... too bad...